src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,57 @@
+import { tagsItems } from '@constants';
+import classes from './Footer.module.scss';
+
+const { footer, contacts, contacts__column, contacts__header } = classes;
+const { contacts__contact, tags, tags__header, tags__list } = classes;
+const { tags__tag, copyright } = classes;
+
+function Footer() {
+  return (
+    <footer className={footer}>

```

Comment:
You can divide fooder into small components, coz when you're gonna add more logic, this component will be really huge.